jekyll/_includes/hits-target.html

```diff
@@ -1,7 +1,7 @@
 <div id="hits-target" style="display:none;">
   <div class="container-semi-fluid">
     <div id="subnav-placeholder">
-      <div class="component subnav dynamic-fixed" data-top-nav-offset="120">
+      <div class="component subnav dynamic-fixed" data-top-nav-offset="68">
```

Comment:
Looking into this. I'm going to try making the header element sticky instead of fixed so it just always takes up the correct amount of space in the layout flow.